### PR TITLE
Small updates to relval workflows

### PR DESCRIPTION
**Pull request by fabozzi**
- Adjusted job splitting for SingleElectron 2017 data relval
- Introduced a "Prod" premix workflow for 2018 MC (i.e. same configuration as in MC production)

### CODE REVIEW

**Comment by fabozzi** [comment]
please test workflow 250200.118

**Comment by fabozzi** [comment]
+1

**Comment by kpedro88** [comment]
+1

**Comment by fabiocos** [comment]
@fabozzi all the DQM changes are in the premixing workflow 250202.181. Are they expected from the modifications of the RECOPRMX18PROD setup? Furthermore, the new wf 250202.118 seems to miss an HARVESTING step, is this the expected behaviour? I doubt...

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -499,15 +504,15 @@ def gen2017(fragment,howMuch):
...
-steps['SMS-T1tttt_mGl-1500_mLSP-100_13UP18']=gen2017('SMS-T1tttt_mGl-1500_mLSP-100_13TeV-pythia8_cfi',Kby(9,50))
+steps['NuGun_UP18']=gen2018('SingleNuE10_cfi.py',Kby(9,50))
+steps['TTbar_13UP18']=gen2018('TTbar_13TeV_TuneCUETP8M1_cfi',Kby(9,50))
+steps['ProdZEE_13UP18']=gen2018('ZEE_13TeV_TuneCUETP8M1_cfi',Kby(9,50))
+steps['ZEE_13UP18']=gen2018('ZEE_13TeV_TuneCUETP8M1_cfi',Kby(9,50))
+steps['ZMM_13UP18']=gen2018('ZMM_13TeV_TuneCUETP8M1_cfi',Kby(18,100))
+steps['ZTT_13UP18']=gen2018('ZTT_All_hadronic_13TeV_TuneCUETP8M1_cfi',Kby(9,80))
+steps['H125GGgluonfusion_13UP18']=gen2018('H125GGgluonfusion_13TeV_TuneCUETP8M1_cfi',Kby(9,50))
+steps['QQH1352T_13UP18']=gen2018('QQH1352T_13TeV_TuneCUETP8M1_cfi',Kby(9,50))
+steps['SMS-T1tttt_mGl-1500_mLSP-100_13UP18']=gen2018('SMS-T1tttt_mGl-1500_mLSP-100_13TeV-pythia8_cfi',Kby(9,50))
```
Comment (makortel) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: @fabiocos This change looks something that could change premixing results.

**Comment by fabozzi** [comment]
@fabiocos 
1. The updated generation step uses 2018 configuration, which is the right thing to do since they are 2018 workflows (I just realized when doing this PR they were still setup to 2017). Since the input gensim of the 250202.181 is updated to 2018, then I do expect changes affecting the workflow.

2. This behaviour is what we want: it is not only missing the harvesting but also DQM, since in MC production we do not have a DQM sequence

**Comment by fabiocos** [comment]
@fabozzi @makortel yes, I indeed see differences in the step1 output that justify modifications. Concerning the second point I see your argument, it is similar to wfs 1 , 2 or 3.

**Comment by fabiocos** [comment]
+1

**Pull request merged**